packages/security/src/components/ProfileImage/ProfileImage-story.js

```diff
@@ -13,15 +13,38 @@ import { components } from '../../../.storybook';
 import { ProfileImage } from '../..';
 
 import { className, large, profile } from './_mocks_';
+import { InlineNotification, NotificationActionButton } from '../../';
 
 const profileImageProps = () => ({
   className,
   large: boolean('Large (large)', large),
 });
 
-storiesOf(components('ProfileImage#legacy'), module).add('default', () => (
-  <ProfileImage
-    {...profileImageProps()}
-    profile={object('Profile (profile)', profile)}
-  />
-));
+storiesOf(components('ProfileImage#legacy'), module)
+  .addDecorator((Story) => (
+    <>
+      <InlineNotification
+        className="page-layouts__banner"
+        actions={
+          <NotificationActionButton
+            href="https://carbon-for-ibm-products.netlify.app/?path=/docs/ibm-products-patterns-user-profile-images-userprofileimage--default"
```

Comment:
```suggestion
            href="https://carbon-for-ibm-products.netlify.app/?path=/story/ibm-products-patterns-user-profile-images-userprofileimage--default"
```

Can we change this link to the story instead of docs?